### ISSUE TITLE
[Public] [Asculta] The screenreader does not read the entire content of the page

### ISSUE THREAD

**Issue by mirunaioana-msc** [code4romania/adapt-just-front#11]
**Describe the bug**
The screenreader only reads some labels/texts. For instance in the page Informatii/Despre violenta, it only read "Despre violenta" and "Cere ajutor"

**Expected behavior**
The screenreader should behave just like on the mobile app and it should read all the information.

**Comment by jacobcatalin** [comment]
this functionality is working now on staging environment.
